CUDADataFormats/BeamSpot/src/BeamSpotCUDA.cc

```diff
@@ -4,5 +4,5 @@
 
 BeamSpotCUDA::BeamSpotCUDA(Data const* data_h, cuda::stream_t<>& stream) {
   data_d_ = cudautils::make_device_unique<Data>(stream);
-  cuda::memory::async::copy(data_d_.get(), data_h, sizeof(Data), stream.id());
+  cudaMemcpyAsync(data_d_.get(), data_h, sizeof(Data), cudaMemcpyHostToDevice, stream.id());
```

Comment:
We've typically used `cudaMemcpyDefault` elsewhere (but I'm not against of denoting the direction explicitly).

Reply:
So could I leave the ```cudaMemcpy``` with the direction defined?

Reply:
> So could I leave the `cudaMemcpy` with the direction defined?

I'm fine with that.